script.js

```diff
@@ -0,0 +1,89 @@
+
+var tiles = document.querySelectorAll(".tile")
+
+var moves = 0
```

Comment:
I'd recommend to keep up with this practice of declaring variables at the top of files. It helps with organization.